### DISCUSSION TITLE
Set a video wallpaper based on time or random from a list.

### DISCUSSION THREAD

**Discussion by Knives79** [rocksdanister/lively#240] [Ideas]
Hello, I want to propose my idea of being able to change wallpapers in two automatic modes.
The first using for example the time of day to set a different wallpaper, the second, instead set a random wallpapers every few minutes chosen from the library (like windows with static ones).
Thanks for the great work done to date, best regards.

Marco

**Comment by rocksdanister** [comment]
Playlist covers these cases https://github.com/rocksdanister/lively/issues/137
